docs/miscellaneous.rst

```diff
@@ -8,6 +8,8 @@ Miscellaneous
 Layout of State Variables in Storage
 ************************************
 
+.. _storage-inplace-encoding:
```

Comment:
Did you test these? As far as I know, they have to come before the header they refer to

Reply:
Yes, that works. I meant to refer to that specific paragraph and not the entire section.